build/vsts_build.yaml

```diff
@@ -75,13 +92,23 @@ phases:
 
   - task: MSBuild@1
     displayName: "Run unit tests"
+    continueOnError: "true"
     inputs:
       solution: "build\\build.proj"
       msbuildVersion: "15.0"
       configuration: "$(BuildConfiguration)"
       msbuildArguments: "/t:CoreUnitTests;UnitTestsVS15 /p:NUGET_PFX_PATH=$(Build.Repository.LocalPath)\\keys\\NuGetKey.snk /p:MS_PFX_PATH=$(Build.Repository.LocalPath)\\keys\\35MSSharedLib1024.snk  /p:BuildRTM=$(BuildRTM) /p:BuildNumber=$(Revision) /p:TestResultOutputFormat=xml"
     condition: "and(succeeded(),eq(variables['BuildRTM'], 'true'))"
 
+  - task: PowerShell@1
+    displayName: "Initialize Git Commit Status on GitHub"
```

Comment:
this seems identical to the previous display name.  Can we please rename this to be different per test?

Reply:
the display name is the task it's doing , regardless of which kind of test it is doing it for.